src/extension/commands/test/runUITests.ts

```diff
@@ -27,20 +27,22 @@ export default class RunUITests extends ReactNativeAppCommand {
             return false;
         }
 
-        // Only iOS is supported for now
-        if (app.os.toLowerCase() !== AppCenterOS.iOS.toLowerCase()) {
-            VsCodeUtils.ShowWarningMessage(Strings.OnlyIOSError);
-            return false;
-        }
-
-        const testRunner = new IOSTestRunner({
+        let testRunner;
+        const platformDir = app.os.toLowerCase() === AppCenterOS.iOS.toLowerCase() ? ReactNativePlatformDirectory.IOS : ReactNativePlatformDirectory.Android;
```

Comment:
Can we save `app.os.toLowerCase() === AppCenterOS.iOS.toLowerCase()` to e.g. isIOSPlatform var so we can use this var also below (line 41)?